narwhal/node/tests/reconfigure.rs

```diff
@@ -197,7 +199,11 @@ async fn restart() {
 
         let committee = committee.clone();
         let worker_cache = worker_cache.clone();
-        let parameters = parameters.clone();
+
+        let mut parameters = parameters.clone();
```

Comment:
Would it make sense to just generate a new set of parameters for each node instead of cloning here? Then you would not need to manually acquire a new admin port

Reply:
Ah yes! Thanks @arun-koshy  - good point!